cmd/kubernetes-discovery/artifacts/core-apiservices/legacy.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apiregistration.k8s.io/v1alpha1
+kind: APIService
+metadata:
+  name: v1.
+spec:
+  version: v1
+  service:
+    namespace: default
+    name: kubernetes
+  insecureSkipTLSVerify: true
```

Comment:
Let me have this for now and we'll adjust as we get the serving certs pre-generated.